src/views/annual-report/2020/l10n.json

```diff
@@ -0,0 +1,379 @@
+    "annualReport.2020.EDMessageText4": "In the wake of the pandemic, some have called for a “return to normal.” But for many young people, the freedom to learn and explore was missing in our schools well before COVID-19.",
+    "annualReport.2020.EDMessagePullQuote": "We must do everything we can to change the systemic inequities in our educational systems, because “normal” was not built to be fair and equitable for most of our children.",
+    "annualReport.2020.EDMessageText5": "In 2021, Scratch is redoubling our efforts to reach young people who have been historically excluded from creative computing and other creative learning opportunities. With support from Google.org, we’ve launched the Scratch Education Collaborative (SEC), a powerful network of organizations across the world focused on supporting these learners in developing their confidence in creative computing. The 41 organizations in year one of the new program will connect with and learn from the Scratch Team and one another, and develop Equity Toolkits that will support them as they grow and scale their support for the learners in their community.",
+    "annualReport.2020.EDMessageText6": "Our work to make Scratch even more equitable and inclusive is far from over. I’m excited to share more with you in the coming months. Until then, I’d like to extend my sincere appreciation to the Scratch Community for continuing to support and care for each other through a turbulent year. Your creativity and compassion never ceases to inspire us.",
+
+    "annualReport.2020.EDName": "Shawna Young",
+    "annualReport.2020.EDTitle": "Executive Director, Scratch Foundation",
+
+    "annualReport.2020.lookingForward": "Looking Forward",
+    "annualReport.2020.lookingForwardText1": "In 2021, we’re continuing to innovate and collaborate with our partners to make Scratch even better for young people around the world. In the coming months, we’re working to bring Scratch into more schools, expand pathways to creative learning, develop and localize more resources for educators and young people, and improve the Scratch onboarding experience, and even more exciting projects.",
+    "annualReport.2020.lookingForwardText2": "We’ve received generous grants from the LEGO Foundation and Google.org to help expand our global reach, advance our mission, and support this important work. Learn more:",
+    
+    "annualReport.2020.learnMore": "Learn More:",
+    "annualReport.2020.learnMoreLink1": "https://www.legofoundation.com/en/about-us/news/the-lego-foundation-and-scratch-foundation-announce-partnership-to-support-learning-through-play-with-technology-for-millions-of-children-across-the-world/",
```

Comment:
These links shouldn't be in this file as we don't want translators translating them into other languages. They should just be in the source code.